### DISCUSSION TITLE
Can't Install

### DISCUSSION THREAD

**Discussion by x123xx123xx** [alexjustesen/speedtest-tracker#1081] [Q&A]
using unraid apps, henrys just installs and works instantly.
this one doesnt launch at all, installed mysql created a database, got mysql workshop and logged into database fine no issues.
tried this one again using the hosted database info and nada.

Connection.php line 822: SQLSTATE[HY000] getaddrinfo for MySQL-Database failed: Name or service not known.
Connection.php line 65: SQLSTATE[HY000] getaddrinfo for MySQL-Database failed: Name or service not known.
Connection.php line 65: __construct(): getaddrinfo for MySQL-Database failed: Name or service not known.

s6-rc warning unable to start service laravel automations/ failed to properly bring all services up.

**Comment by valboosky** [comment]
i had to go into my database config file and change the binding from 127.0.0.1 to 0.0.0.0
